### PR TITLE
fix: collection description not updating[]

### PR DESCRIPTION
**Pull request by LordNayan**
## Description

Fix - Collection description not updating.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

File: src/lib/components/collections/collection-tab/MyCollection.viewModel.ts [hunk 1]
```diff
@@ -29,40 +29,48 @@ export class MyCollectionViewModel {
...
-          { name: newCollectionName },
-        );
+    const updateObj = {};
 
-      tabName = updateCollectionElement.data.data.name;
-
-      collectionsMethods.updateCollection(
+    switch (property) {
+      case "name": {
+        updateObj.name = value;
```
Comment (parthdalaltech) [inline src/lib/components/collections/collection-tab/MyCollection.viewModel.ts:40]: use same -> updateObj[property] = value;

**Review by parthdalaltech — CHANGES_REQUESTED** [review]

File: src/lib/components/collections/collection-tab/MyCollection.viewModel.ts (hunk 1 above)
Comment (LordNayan) [reply src/lib/components/collections/collection-tab/MyCollection.viewModel.ts:40]: Correct. How did this not come to my mind 🤣

**Review by parthdalaltech — DISMISSED** [review]

**Pull request merged**